storage-proofs-porep/src/stacked/vanilla/proof.rs

```diff
@@ -140,8 +152,7 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
             Challenges::Synth(synth_challenges) => {
                 // If there are no synthetic vanilla proofs stored on disk yet, generate them.
                 if pub_inputs.seed.is_none() {
-                    info!("generating synthetic vanilla proofs in a single partition");
-                    assert_eq!(partition_count, 1);
```

Comment:
Looks good to me! I'm not seeing why this one `assert` was removed, but I won't let it delay approval.

Reply:
Nvm, I'm looking at #1750 now.

Reply:
> Looks good to me! I'm not seeing why this one `assert` was removed, but I won't let it delay approval.

Adding it was an error.  The partition count is only 1 for test sectors in porep, and the logging comment implied the incorrect thing (i.e that the single pass of synth porep proof generation implies that the partition count must be 1, which is wrong; it just meant we were not generating synth porep proofs with regard to challenges being partition specific like in other places).

Reply:
^Fortunately, it was never released with that assert.  It came up quickly in overall testing while looking into this issue though.